Slambook/index.js

```diff
@@ -67,9 +67,32 @@ form.addEventListener('submit',function(e){
     allSlamArray.push(formValue)
     console.log(formValue)
 console.log(allSlamArray)
+showSlamList();
+})
```

Comment:
allSlamArray is not defined within the function. Make sure to pass it as a parameter or define it inside the function.